wunderio/next-drupal-starterkit
# NEX-149: Update to next.js 14.2.1

## Changes proposed in this PR:

* update to next.js 14.2.1
* tweaking of the `./setup.sh` command
* small lando tweak for kibana

## How to test:

1. check the site works on silta
2. run the `setup.sh` command locally
